packages/@ama-terasu/cli/src/helpers/format.helper.ts

```diff
@@ -33,10 +33,10 @@ const formatHelpOptionsBlocks = (initialMessage: string, groups: string[]) => {
           }
 
           return line
-            .substring(0, indexSep)
+            .slice(0, Math.max(0, indexSep))
```

Comment:
Is this more performant than `substring`? I find it complexifying the sintax

Reply:
seems not (on Chrome + Mac at least)
<img width="853" alt="image" src="https://github.com/user-attachments/assets/a8ee8ac3-00fa-417b-9bdf-a4c909f52d98">

cf: https://jsben.ch/bRIhQ

Reply:
rule `unicorn/prefer-string-slice`